app/views/inventory_reconciliations/_incomplete_deltas.html.erb

```diff
@@ -19,16 +19,16 @@
 
       <tbody>
         <% @reconciliation.deltas.each do |delta| %>
-          <% reconciliation_delta_table_row(delta) do %>
-            <td class="<%= delta.description_css_class %>">
+          <%= reconciliation_delta_table_row(delta) do %>
+            <td class="<%= description_css_class(delta) %>">
               <%= delta.item.category.description %> - <%= delta.item.description %>
             </td>
             <td><%= delta.requested_quantity %></td>
             <td><%= delta.current_quantity %></td>
             <td><%= delta.final_count %></td>
 
-            <td class="<%= delta.changed_amount_css_class %>">
-              <%= changed_amount_icon(delta) %>
+            <td class="<%= changed_amount_css_class(delta) %>">
+              <%= tag(:i, class: changed_amount_icon_class(delta)) %>
```

Comment:
Why the change back to the tag here...? Won't this mean we have a dangling `i` tag when the class returns nothing?

Reply:
Also, we are on Rails 5... doesn't that support the new `tag.i class: "xyz"` syntax? Maybe for this and the helper that does the `tr` `content_tag`, we could use the new syntax?